docs/Bots.md

```diff
@@ -1221,6 +1222,31 @@ Parses breaches and pastes and creates one event per e-mail address. The e-mail
 
 * * *
 
+### Key-Value Parser
+
+#### Information:
+* `name:` intelmq.bots.parsers.key_value.parser
+* `lookup:` no
+* `public:` no
+* `cache (redis db):` none
+* `description:` Parses text lines in key=value format.
+
+#### Configuration Parameters:
+
+* `pair_separator`: String separating key=value pairs, default "` `" (space).
+* `kv_separator`: String separating key and value, default `=`.
+* `keys`: Array of string->string, names of keys to propagate mapped to IntelMQ event fields. Example:
+   ```json
+   "keys": {
+       "srcip": "source.ip",
+       "dstip": "destination.ip"
+   }
+   ```
+* `strip_quotes`: Boolean, remove opening and closing quotes from values, default true.
```

Comment:
This is only valid for double quotes (not single quotes annd backticks). Please specify this here.

Reply:
Documentation added.